perun-core/src/main/java/cz/metacentrum/perun/core/api/AuditMessagesManager.java

```diff
@@ -62,7 +62,7 @@ public interface AuditMessagesManager {
 	List<AuditMessage> getMessagesByIdAndCount(PerunSession perunSession, int id, int count) throws PrivilegeException;
 
 	/**
-	 * Returns page of audit messages. Query parameter specifies offset and page size. Total count is only estimated.
+	 * Returns page of audit messages filtered by name of event. Query parameter specifies offset and page size. Total count is only estimated.
```

Comment:
```suggestion
	 * Returns page of audit messages. Query parameter specifies offset and page size and allows filtering  by name of event. Total count is only estimated.
```
nitpick - theoretically the query parameters also includes the filtering.